packages/exchange/migrations/1604562715444-ExchangeTablePrefix.ts

```diff
@@ -0,0 +1,91 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class ExchangeTablePrefix1604562715444 implements MigrationInterface {
+    name = 'ExchangeTablePrefix1604562715444';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `CREATE TABLE "exchange_account" ("createdAt" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), "id" uuid NOT NULL DEFAULT uuid_generate_v4(), "userId" character varying NOT NULL, "address" character varying NOT NULL, CONSTRAINT "PK_6dbfd7bdb34f40b831d3013a57f" PRIMARY KEY ("id"))`
+        );
```

Comment:
We are creating a new set of tables here, could we try `ALTER TABLE table_name
RENAME TO new_table_name;` instead?

Reply:
Good point. This is the autogenerated code by NestJS, not sure why they don't already do it that way.